tests/validation/config.json

```diff
@@ -921,7 +921,8 @@
         },
         {
             "title": "KTX2 decoder test",
-            "playgroundId": "#EIJH8L#1",
+            "playgroundId": "#EIJH8L#26",
+            "renderCount": 5,
```

Comment:
Why do you need a render count?

Reply:
I don't remember why, it works without it - I have removed it.